app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'AuthorId'
+  belongs_to :post, class_name: 'Post', foreign_key: 'PostId'
+end
```

Comment:
- [ ] Please add a method that updates the comments counter for a post. This is a project requirement. 👍

Reply:
Thank you